containers/Dockerfile.buildenv

```diff
@@ -108,6 +108,9 @@ RUN wget https://bitbucket.org/blaze-lib/blaze/downloads/blaze-3.2.tar.gz -O bla
     && rm -rf brigand \
     && wget https://github.com/catchorg/Catch2/releases/download/v2.2.1/catch.hpp -O catch.hpp \
     && mv catch.hpp /usr/local/include \
+    && wget https://github.com/pybind/pybind11/archive/v2.4.3.tar.gz -O pybind11.tar.gz \
+    && mv pybind11/include/pybind11 /usr/local/include \
```

Comment:
These should be:

```
    && wget https://github.com/pybind/pybind11/archive/v2.4.3.tar.gz -O pybind11.tar.gz \
    && tar xzf pybind11.tar.gz \
    && mv pybind11-2.4.3/include/pybind11 /usr/local/include \
    && rm -rf pybind11 \
```

Reply:
🤦‍♂️ I should have built the container to check. Doing so now

Reply:
I've pushed the updated container built with exactly the lines I posted.